deploy/skaffold/Dockerfile

```diff
@@ -12,59 +12,91 @@
+ENV KOMPOSE_URL https://github.com/kubernetes/kompose/releases/download/${KOMPOSE_VERSION}/kompose-linux-amd64
+RUN wget -O kompose "${KOMPOSE_URL}"
+RUN chmod +x kompose
+
+# Download container-structure-test
+FROM alpine:3.9 as download-container-structure-test
+ENV CONTAINER_STRUCTURE_TEST_VERSION v1.5.0
+ENV CONTAINER_STRUCTURE_TEST_URL https://storage.googleapis.com/container-structure-test/${CONTAINER_STRUCTURE_TEST_VERSION}/container-structure-test-linux-amd64
+RUN wget -O container-structure-test "${CONTAINER_STRUCTURE_TEST_URL}"
+RUN chmod +x container-structure-test
+
+# Download kind
+FROM alpine:3.9 as download-kind
+ENV KIND_VERSION v0.3.0
+ENV KIND_URL https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-amd64
+RUN wget -O kind "${KIND_URL}"
+RUN chmod +x kind
+
+# Download gcloud
+FROM alpine:3.9 as download-gcloud
+ENV GCLOUD_VERSION 245.0.0
+ENV GCLOUD_URL https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GCLOUD_VERSION}-linux-x86_64.tar.gz
+RUN wget -O gcloud.tar.gz "${GCLOUD_URL}"
+RUN tar -zxvf gcloud.tar.gz
+
+# Download bazel
+FROM alpine:3.9 as download-bazel
+ENV BAZEL_VERSION 0.27.0
+ENV BAZEL_URL https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-x86_64
+RUN wget -O bazel "${BAZEL_URL}"
+RUN chmod +x bazel
+
 FROM gcr.io/gcp-runtimes/ubuntu_16_0_4 as runtime_deps
 
 RUN apt-get update && \
   apt-get install --no-install-recommends --no-install-suggests -y \
-  git \
-  python && \
+  git python unzip && \
   rm -rf /var/lib/apt/lists/*
 
-ENV KUBECTL_VERSION v1.12.0
-RUN curl -Lo /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl  && \
-  chmod +x /usr/local/bin/kubectl
-
-ENV HELM_VERSION v2.8.1
-RUN curl -LO https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz && \
-  tar -xvf helm-${HELM_VERSION}-linux-amd64.tar.gz -C /usr/local/bin --strip-components 1 && \
-  rm -f helm-${HELM_VERSION}-linux-amd64.tar.gz
-
-ENV CLOUD_SDK_VERSION 217.0.0
-RUN curl -LO https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-  tar -zxvf google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
-  CLOUDSDK_PYTHON="python2.7" /google-cloud-sdk/install.sh --usage-reporting=false \
-  --bash-completion=false \
-  --disable-installation-options && \
-  rm -rf google-cloud-sdk-*.tar.gz
+COPY --from=docker:18.09.6 /usr/local/bin/docker /usr/local/bin/
+COPY --from=download-kubectl kubectl /usr/local/bin/
+COPY --from=download-helm helm /usr/local/bin/
+COPY --from=download-kustomize kustomize /usr/local/bin/
+COPY --from=download-kompose kompose /usr/local/bin/
+COPY --from=download-container-structure-test container-structure-test /usr/local/bin/
+COPY --from=download-kind kind /usr/local/bin/
+COPY --from=download-bazel /bazel /usr/local/bin/
```

Comment:
just to be consistent can this be `bazel` as well instead of `/bazel` or all the above `/xyz`?

Reply:
or I guess for directories this is fine?

Reply:
same for google-cloud-sdk

Reply:
done!